### ISSUE TITLE
Python requests.get() throws an exception in aws lambda

### ISSUE THREAD

**Issue by anil-grexit** [serverless/serverless#6801]
I'm not able to make any HTTP calls from python-based lambda function.

```python
import os
import requests

def verify_token(token, smId):
    url = os.getenv('V2_URL') + '/api/analytics/validate/' + smId
    headers = {
        'Content-Type':'application/json',
        'Authorization': token
    }

    result = requests.get(url, headers=headers)

    print('Result: ')
    print(result.content)
```

In Cloudwatch, I see this stack trace:

```python
[ERROR] UnboundLocalError: local variable 'response' referenced before assignment
Traceback (most recent call last):
  File "/var/task/serverless_sdk/__init__.py", line 97, in wrapped_handler
    return user_handler(event, context)
  File "src/authorize.py", line 21, in validate
    principal_id = verify_token(whole_auth_token, event['pathParameters']['smId'])
  File "src/authorize.py", line 38, in verify_token
    response = requests.get(url, headers=headers)
  File "/var/task/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/var/task/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/var/task/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/var/task/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/var/task/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/var/task/serverless_sdk/__init__.py", line 384, in wrapper
    if response:
```

**Comment by dschep** [comment]
fixed in `@serverless/enterprise-plugin` 3.1.2. Upgrade with `npm i -g serverless` and verify the plugin version with `sls --version`.

**Comment by shadiakiki1986** [comment]
Just want to add that I needed to re-deploy the code after the upgrade
